<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>168</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teams</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lTeam">
     <property name="text">
      <string>Team</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lWins">
     <property name="text">
      <string>Wins</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="lLosses">
     <property name="text">
      <string>Losses</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="leTeam"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="leWins">
     <property name="inputMask">
      <string>000</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="leLosses">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="inputMask">
      <string>000</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lDraws">
     <property name="text">
      <string>Draws</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lGoals">
     <property name="text">
      <string>Goals</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lGoalsA">
     <property name="text">
      <string>Goals Against</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLineEdit" name="leDraws">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="inputMask">
      <string>000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="leGoals">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="inputMask">
      <string>000</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="leGoalsA">
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
     <property name="inputMask">
      <string>000</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnClean">
     <property name="text">
      <string>Clean</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leTeam</tabstop>
  <tabstop>leWins</tabstop>
  <tabstop>leLosses</tabstop>
  <tabstop>leDraws</tabstop>
  <tabstop>leGoals</tabstop>
  <tabstop>leGoalsA</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnClean</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnSearch</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>callFunctionTeam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClean</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>clearAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSearch</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>searchTeam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>callFunctionTeam()</slot>
  <slot>clearAll()</slot>
  <slot>searchTeam()</slot>
 </slots>
</ui>
